addons/mail/static/src/composer/navigable_list.js

```diff
@@ -63,9 +63,6 @@ export class NavigableList extends Component {
     }
 
     open() {
-        if (this.state.isLoading) {
```

Comment:
I feel like it is more like a work around to the fact the promise we're awaiting for the "loading" state to end is never resolved and replaced by other promises. I'm not sure if we should care though.